R/fit.R

```diff
@@ -937,32 +937,36 @@ CmdStanMCMC <- R6::R6Class(
       private$metadata_ <- data_csv$metadata
 
       if (!is.null(data_csv$post_warmup_draws)) {
+        missing_variables <- !(posterior::variables(data_csv$post_warmup_draws) %in% posterior::variables(private$draws_))
```

Comment:
Is this stuff related to some other issue besides the trailing slash problem?

Reply:
Ah sorry, that should be a part of https://github.com/stan-dev/cmdstanr/pull/293
I had both on the same branch then split them up.